arangodb-net-standard/AqlFunctionApi/Models/PostExplainAqlQueryResponseNode.cs

```diff
@@ -9,14 +9,14 @@ namespace ArangoDBNetStandard.AqlFunctionApi.Models
     public class PostExplainAqlQueryResponseNode
     {
         public string Type { get; set; }
-        public IList<int> Dependencies { get; set; }
+        public IEnumerable<int> Dependencies { get; set; }
```

Comment:
From what I understand, this is a breaking change. Any change from `IList<T>` to `IEnumerable<T>` in response models is a breaking change. (e.g. see the tests updates, `Count` vs `Count()`).

The following code won't compile after the changes:
```
IList<string> projections = response.Projections; // CS0266 error
```

```
var projections = response.Projections;

// Anyting using IList<T> methods will cause errors, e.g. CS0021.
```

❓ Is it on purpose?

It's mostly beneficial for us, the library developers (or for customer implementation of serialization), to use `IEnumerable<T>` in response models, as we can change the implementation without breaking existing code. But honestly, I don't think it's worth introducing a breaking change, though it's a fairly minor one.

❓  Should we revert the change for response model and schedule it in a major release?

Reply:
Semantic versioning doesn't say anything about the size of the update, so bumping to 3.0 should be fine I think.